docs/devguide/hexformat.rst

```diff
@@ -67,3 +105,48 @@ Running the ``make all`` command executes the following steps:
 - The ``tools/adduicr.py`` script adds the UICR to the bare hex
 - The final hex file is placed in ``build/firmware.hex``
 - The user can optionally append a script using ``tools/makecombinedhex.py`` (or other tools)
+
+micro:bit V2
+============
+
+This applies to MicroPython for the micro:bit V2, the source of which can be found here: `microbit-foundation/micropython-microbit-v2 <https://github.com/microbit-foundation/micropython-microbit-v2>`_.
+
+This is a port of MicroPython to the micro:bit which uses the CODAL as the underlying target platform.
+
+After cloning this repository update the submodules:
+
+`$ git submodule update --init`
+
+Then build the MicroPython cross-compiler:
+
+`$ make -C lib/micropython/mpy-cross`
+
+After setting up, go to the src/ directory and build:
+
+`$ cd src`
+
+`$ make`
+
+That will build both libmicropython.a (from source in src/codal_port/) and the CODAL app (from source in src/codal_app/). The resulting firmware will be MICROBIT.hex in the src/ directory which can be copied to the micro:bit.
```

Comment:
This is just a nitpick, but it looks like filenames and paths have been added in code quotes:

```suggestion
That will build both ``libmicropython.a`` (from source in ``src/codal_port/``) and the CODAL app (from source in ``src/codal_app/``). The resulting firmware will be ``MICROBIT.hex`` in the ``src/`` directory which can be copied to the micro:bit.
```

Reply:
There are more in the doc that will need the quotes as well.